app/web/components/EllipsisMenu.tsx

```diff
@@ -0,0 +1,86 @@
+import { MoreHoriz } from "@mui/icons-material";
+import { IconButton, Menu, styled } from "@mui/material";
+
+interface EllipsisMenuProps {
+  children: React.ReactNode;
+  idName: string;
+  isMenuOpen: boolean;
+  menuAnchorEl: Element | null;
+  onMenuOpen: (event: React.MouseEvent<HTMLButtonElement>) => void;
+  onMenuClose: (event: React.MouseEvent<HTMLButtonElement>) => void;
+}
+
+const MenuWrapper = styled("div")(({ theme }) => ({
+  display: "flex",
+  justifyContent: "flex-end",
+  flexDirection: "column",
+}));
+
+//** @param {children} should be the  MenuItems */
+const EllipsisMenu = ({
+  children,
+  idName,
+  isMenuOpen,
+  menuAnchorEl,
+  onMenuOpen,
+  onMenuClose,
+}: EllipsisMenuProps) => {
```

Comment:
I already know I want to use this for block functionality too so making it's own component. And we can switch out the menu in NotificationFeed instead of redoing it every time.